src/apm.coffee

```diff
@@ -61,7 +61,7 @@ module.exports =
     process.env.ATOM_API_URL ? 'https://atom.io/api'
 
   getNodeVersion: ->
-    process.env.ATOM_NODE_VERSION ? '0.22.0'
+    process.env.ATOM_NODE_VERSION ? '0.22.3'
```

Comment:
It looks like it has to be updated as it's used to build/search native modules: 
![screenshot 1](https://cloud.githubusercontent.com/assets/25722/9334908/23fca760-45c9-11e5-89c3-617490525080.png)
(`serialport` module was installed using `Update Package Dependencies: Update` command)